### PR TITLE
chore: bump dependencies

### PR DESCRIPTION
**Pull request by utkuozdemir**
Bump pkgs version and the dependencies.

Signed-off-by: Utku Ozdemir <utku.ozdemir@siderolabs.com>

### CODE REVIEW

File: nvidia-gpu/vars.yaml [hunk 1]
```diff
@@ -8,10 +8,10 @@ CONTAINER_TOOLKIT_REF: d9de4a09b8fd51a46207398199ecfeb3998ad49d
 LIBNVIDIA_CONTAINER_VERSION: v1.11.0
 LIBNVIDIA_CONTAINER_REF: c8f267be0bac1c654d59ad4ea5df907141149977
 # renovate: datasource=github-tags extractVersion=^go(?<version>.*)$ versioning=loose depName=golang/go
-GOLANG_VERSION: 1.17.13
```
Comment (smira) [inline nvidia-gpu/vars.yaml:11]: only @frezbo knows whether it's safe to update :see_no_evil:

**Review by smira — APPROVED** [review]

File: nvidia-gpu/vars.yaml (hunk 1 above)
Comment (frezbo) [reply nvidia-gpu/vars.yaml:11]: yeh, we need to check if there's a new libnvidia-container update, then cross check with their makefiles. Also if the code compiles i think it's safe to update once we complete the nvidiai e2e tests.

File: nvidia-gpu/nvidia-container-toolkit/nvidia-container-cli/pkg.yaml [hunk 2]
```diff
@@ -62,6 +62,9 @@ steps:
 
         patch -p0 < /pkg/patches/libnvidia-container/Makefile.patch
 
+        # workaround for Debian 11 (Bullseye): https://github.com/NVIDIA/libnvidia-container/issues/116#issuecomment-826146121
+        make || true
+        mv deps/src/elftoolchain-0.7.1/libelf/'name libelf.so.1' deps/src/elftoolchain-0.7.1/libelf/libelf.so.1
```
Comment (frezbo) [inline nvidia-gpu/nvidia-container-toolkit/nvidia-container-cli/pkg.yaml:67]: i've seen this potential fix, maybe we could build libelf first, then build the actual one :thinking:

**Review by frezbo — APPROVED** [review]

**Pull request merged**